Makefile.am

```diff
@@ -72,6 +72,8 @@ BUILT_SOURCES = \
 	$(NULL)
 
 eks_search_provider_v2_SOURCES = \
+	search-provider/ekn-query-util.c \
+	search-provider/ekn-query-util.h \
```

Comment:
Any reason not to use `eks-` for these files?

Reply:
I could - only reason why I used ekn was that they were just helper functions for ekn queries. I'll change the filename :)